### PR TITLE
Need to also push the latest tag.

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:+1:

File: demo/Makefile [hunk 1]
```diff
@@ -25,3 +25,4 @@ push:
 
 push-latest: push
```
Comment (tfennelly) [inline demo/Makefile]: So this target is going to push 2 images?

**Comment by tfennelly** [comment]
:+1:

File: demo/Makefile (hunk 1 above)
Comment (jglick) [inline demo/Makefile]: Well, one image with two tags. Currently

```
$ curl -s https://registry.hub.docker.com/v1/repositories/jenkinsci/workflow-demo/tags | json_pp
[
   {
      "name" : "latest",
      "layer" : "6b64b99c"
   },
   {
      "layer" : "d270500e",
      "name" : "1.5"
   },
   {
      "layer" : "6b64b99c",
      "name" : "1.6"
   },
   {
      "name" : "1.6-alpha-1",
      "layer" : "956a5bcd"
   },
   {
      "name" : "1.7-alpha-1",
      "layer" : "9d6e971f"
   }
]
```

**Pull request merged**